Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,23 @@
 import React from 'react';
 import './Header.css';
+import moment from 'moment';
+
+const date = new Date();
+const format = 'D MMM';
+const time = moment(date).format(format);
+const currentDate = time.toLowerCase();
+
+function HeaderTitle (props) {
+    return (
+        <div>
+            <div className="header-title">
+                <h4>{props.title}</h4>
+                <p className='small-text'>@{props.title}</p>
+                <p class='small-text'>•</p>
+                <p className='small-text'>{currentDate}</p>
+            </div>
```

Comment:
great use of Moment for this portion, you could also do it inline as Moment has some great react functionality that you can look into as well.